### ISSUE TITLE
Support serving MPEG-TS recordings with playback server

### ISSUE THREAD

**Issue by aler9** [bluenviron/mediamtx#3054]
## Describe the feature

Discussed in #3045

**Comment by iPixelGalaxy** [comment]
Yes please, this would be super nice to have so I can stream my content to VRChat players on Quest Standalone!

**Comment by pikachu937** [comment]
Greetings.
It would be just great!
Also, if it’s not difficult, I ask you to add the ability to play mpegts on apple smartphones/tablets/computers. By default they are not played, so I have to convert via ffmpeg, which is not very convenient.

**Comment by pikachu937** [comment]
Perhaps if you fix [this](https://github.com/bluenviron/mediamtx/issues/3206), you will be able to send mpegts as a playlist in the playback service

**Comment by pikachu937** [comment]
I rechecked the playlist generation in my service and found an error. Files generated by the recording function in mediamtx can be used in HLS playlists as static ones (that is, they can be sent to playback in the form of an HLS playlist). However, one of the problems still remains: the files cannot be read by the native player in iOS/MacOS, but I wrote about this in another issue

**Comment by maxlandon** [comment]
Hello,

Any plans to implement MPEG-TS playback support ?
